packages/console-extension/schema/tracker.json

```diff
@@ -68,6 +68,16 @@
         "command": "console:restart-kernel",
         "selector": ".jp-CodeConsole",
         "rank": 30
+      },
+      {
+        "command": "console:undo",
+        "selector": ".jp-CodeConsole-promptCell",
+        "rank": 1
```

Comment:
More for consistency: should we put these to the top of the list so they are ordered by rank?